Build.cmd

```diff
@@ -109,11 +109,6 @@ REM Run unit tests (9009 means XUnit itself wasn't found, which is an error).
 xunit.console.x86 Program\Tests.dll -nunit xunit.xml
 IF "%ERRORLEVEL%" == "9009" GOTO :error
 
-CALL :copy "Program\RunActivity.exe" "Program\RunActivityLAA.exe" || GOTO :error
-editbin /NOLOGO /LARGEADDRESSAWARE "Program\RunActivityLAA.exe" || GOTO :error
```

Comment:
I think you can remove the `editbin` check at the start of `Build.cmd` as this is the only use of it

Reply:
Hi James,

Good catch

Chris